python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -179,16 +179,17 @@ def createTempExp(self, calexpRefList, skyInfo, visitId=0):
                 if numGoodPix > 0 and not didSetMetadata:
                     coaddTempExp.setCalib(exposure.getCalib())
                     coaddTempExp.setFilter(exposure.getFilter())
+                    # PSF replaced after loop with CoaddPsf if and only if creating direct warp
```

Comment:
To avoid confusion (which I was subject to at first!), perhaps rephrase as "PSF replaced with CoaddPsf after loop..."